### PR TITLE
Add Pipe Method

### PR DESCRIPTION
**Pull request by koaning**
Added a simple `pipe` method. Fixes https://github.com/RasaHQ/whatlies/issues/130

### CODE REVIEW

File: whatlies/embeddingset.py [hunk 1]
```diff
@@ -181,6 +181,15 @@ def compare_against(self, other, mapping="direct"):
         if mapping == "direct":
             return [v > other for k, v in self.embeddings.items()]
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Applies a function to the embedding set. Useful for method chaining.
+        :param func: The function to apply
```
Comment (mkaze) [inline whatlies/embeddingset.py:187]: Has the style of docstrings changed? E.g. we didn't use `:param` previously?!

Comment (koaning) [reply whatlies/embeddingset.py:187]: Well spotted. Will fix this!

File: whatlies/embeddingset.py [hunk 2]
```diff
@@ -181,6 +181,38 @@ def compare_against(self, other, mapping="direct"):
         if mapping == "direct":
             return [v > other for k, v in self.embeddings.items()]
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Applies a function to the embedding set. Useful for method chaining and
+        chunks of code that repeat.
+
+        Arguments:
+             func: callable that accepts an embedding set as it's first argument
```
Comment (mkaze) [inline whatlies/embeddingset.py:190]: an **\`EmbeddingSet\`** as **its** first argument

Comment (koaning) [reply whatlies/embeddingset.py]: Again, well spotted!

**Comment by koaning** [comment]
@mkaze can I merge this?

**Comment by mkaze** [comment]
Sure, looks good to me!

**Pull request merged**